### ISSUE TITLE
fasd:2: fork failed: resource temporarily unavailable

### ISSUE THREAD

**Issue by SharkIng** [robbyrussell/oh-my-zsh#4743]
When i trying to use fasd plugins, it give me this error message. however, when i remove plugins(.... fasd), it works well but i cannot use fasd. How to solve this porblem?

**Comment by mcornella** [comment]
Is this still happening? If so, tell us more about your system (operating system and version, `zsh --version`, your zshrc file, ...).

**Comment by raphaelfeng** [comment]
I had the same problem and I found I define a function which is the example from the fzf project.
 v() {
    # local file
    # file="$(fasd -Rfl "$1" | fzf -1 -0 --no-sort +m)" && vi "${file}" || return 1
 }

And I also have an alias defined as
alias v="fasd -e vim"
which is an example from the fasd project.

The problem was solved by removing either of them.
I guess the same name caused some recursive calls which run out of the process resource.

**Comment by rossmckelvie** [comment]
I ran into this same problem when sourcing a bunch of aliases/functions where one of my functions had the same name as an alias in the git plugin. Even with the git plugin removed from my zshrc, I was still unable to run any git commands until we renamed our function. Since then, the problem has vanished for me and my coworkers.

tl;dr: check that you did not add a function or alias that collides with something in oh-my-zsh plugins.
